presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -106,6 +107,7 @@
     private final ExecutionPolicy executionPolicy;
     private final List<Expression> parameters;
     private final SplitSchedulerStats schedulerStats;
+    private final AtomicReference<ResourceGroupId> resourceGroupIdAtomicReference = new AtomicReference<>();
```

Comment:
It's all the information that's required to understand what the contents of this variable are. The fact that it's an id is not super relevant in this context, but `resourceGroupId` would be a reasonable name. We expect people to use an IDE to edit the code in most cases, so type information is readily available. We don't particularly care that it's an atomic reference in most of it's usages, and it makes the code much less readable.